eclair-core/src/main/scala/fr/acinq/eclair/io/PeerEvents.scala

```diff
@@ -31,4 +31,6 @@ case class PeerDisconnected(peer: ActorRef, nodeId: PublicKey) extends PeerEvent
 
 case class LastChannelClosed(peer: ActorRef, nodeId: PublicKey) extends PeerEvent
 
+case class receivedPong(latency: FiniteDuration, nodeId: PublicKey) extends PeerEvent
```

Comment:
nit: to better match the rest of the codebase:

```suggestion
case class PongReceived(nodeId: PublicKey, latency: FiniteDuration) extends PeerEvent
```

Reply:
if you change the order in the event, you need to change the publush line too in PeerConnection.scala

Reply:
Yes, that's what I expect you to do when fixing this comment :)

Reply:
Also, you haven't properly imported `FiniteDuration`, the build is failing, that should be fixed.